src/System.Private.Uri/src/System/UriExt.cs

```diff
@@ -499,29 +499,46 @@ internal unsafe bool InternalIsWellFormedOriginalString()
         public static string UnescapeDataString(string stringToUnescape)
         {
             if ((object)stringToUnescape == null)
+            {
```

Comment:
>{ [](start = 12, length = 1)

fine either way to add braces but just FYI in the specific case of a single line block that is not nested in another block we allow omitting braces

[https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/coding-style.md](https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/coding-style.md)